pages/common/phpcs.md

```diff
@@ -0,0 +1,35 @@
+# phpcs
+
+> Tokenize PHP, JavaScript and CSS files to detect violations of a defined set of coding standards.
+
+- Sniff the specified directory for issues:
```

Comment:
What is the default coding standard it checks against ?

Reply:
It defaults to the PEAR standard.

Reply:
Ok, perhaps we can specify that in brackets to make it extra clear. It was a natural question that came to my mind.

Reply:
Makes sense. :smiley: